ports/espressif/boards/esp32-wrover-dev-cam/mpconfigboard.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2024 Chris Drake, independently providing these changes.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+#define MICROPY_HW_BOARD_NAME       "Espressif ESP32-WROVER-DEV-CAM"
```

Comment:
This board isn't made by Espressif so let's use the name on the board.

```suggestion
#define MICROPY_HW_BOARD_NAME       "Freenove ESP32-WROVER-DEV-CAM"
```

Reply:
Do you know if they made it?
None of my boards say "Freenove" on them, except one that came with a robot dog I bought - the brand of which is Freenove.  As far as I can tell, this is an espressif reference design, manufactured by a bunch of different places?
I've written to Freenove to ask them if they designed this or not - better to get the attribution correct than guess :-)

Reply:
Feel free to make an "Unknown" creator id too. I just don't want to misattribute it.

Reply:
Good call @tannewt !  They wrote back.

I'll convert this PR into a draft while I work fix all the ID codes and perform the re-naming of things.


```
Dear customer,

Thank you for choosing Freenove. We are fully responsible for any concerns.

The one with our logo on it is ours, which is the original design.
The ones without logo are counterfeit.

If you have any concerns, please feel free to contact us.

Have a great day!

Best regards,
Shaynee
Freenove Support Team
--------------------------
www.freenove.com 
```